generator/server/impl.js

```diff
@@ -66,28 +88,30 @@ module.exports = {
           return [];
         }
 
+      } else {
+        return [mockDeclarableSprint(1337)];
       }
     }
   },
 
-  addSprint: function (params, body, query, headers) {
+  addSprint: function(params, body, query, headers) {
     body.id = sprints.length + 1;
     sprints.push(body);
     return body;
   },
 
-  getUserAvailability: function (params, body, query, headers) {
-    return Object.assign({}, userAvailability, {userId: null});
+  getUserAvailability: function(params, body, query, headers) {
+    return Object.assign({}, userAvailability, { userId: null });
   },
 
-  updateUserAvailability: function (params, body, query, headers) {
```

Comment:
To nie jest super ważne, ale nie za bardzo mi się podoba, że wszystko jest przeformatowane inaczej nagle - specjalnie jest formater ustawiony, żeby w kółko nie zmieniać sobie nawzajem (wiem, że nie jesteś team IntelliJ, ale jak pytałam na początku, czy ktoś planuje pisać w czymś innym, to nikt nie zaprotestował)